pkgs/tools/filesystems/xfs_undelete/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper
+, coreutils, tcl-8_6, tcllib
+}:
```

Suggestion: This would make the code a bit more diff friendly

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, makeWrapper
, coreutils
, tcl-8_6
, tcllib
}:
```